com.valvesoftware.Steam.appdata.xml

```diff
@@ -3,12 +3,35 @@
 <application>
   <id type="desktop">com.valvesoftware.Steam.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
-  <project_license>LicenseRef-proprietary</project_license>
-  <summary>Application for managing and playing games on Steam</summary>
+  <name>Steam</name>
+  <summary>Manage and play games distributed by Steam</summary>
   <description>
     <p>
-      Steam is an appstore and manager for games.
+      Steam is a popular platform for buying, downloading, and playing
+      video games, and chatting with other players.
     </p>
   </description>
-  <url type="homepage">http://store.steampowered.com/</url>
+  <categories>
+    <category>Game</category>
+    <category>PackageManager</category>
+  </categories>
+  <url type="homepage">https://store.steampowered.com/</url>
+  <url type="help">https://help.steampowered.com/</url>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Valve Corporation</developer_name>
+  <screenshots>
+    <screenshot type="default">
+      <caption>The main Steam library window</caption>
+      <image type="source" width="1280" height="720">https://raw.githubusercontent.com/flathub/com.valvesoftware.Steam/master/com.valvesoftware.Steam.screenshot-0.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Steam Store interface</caption>
+      <image type="source" width="1280" height="720">https://raw.githubusercontent.com/flathub/com.valvesoftware.Steam/master/com.valvesoftware.Steam.screenshot-1.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Steam community</caption>
+      <image type="source" width="1280" height="720">https://raw.githubusercontent.com/flathub/com.valvesoftware.Steam/master/com.valvesoftware.Steam.screenshot-2.png</image>
+    </screenshot>
+  </screenshots>
+  <update_contact>https://github.com/flathub/com.valvesoftware.Steam/issues</update_contact>
```

Comment:
The spec says this needs to be an email, feel free to make it `tingping_AT_fedoraproject.org`